tests/application/test_authorization.py

```diff
@@ -19,14 +19,17 @@
 from ape.utils import ZERO_ADDRESS
 from web3 import Web3
 
+OPERATOR_CONFIRMED_SLOT = 1
 AUTHORIZATION_SLOT = 3
 DEAUTHORIZING_SLOT = 4
 END_DEAUTHORIZATION_SLOT = 5
 MIN_AUTHORIZATION = Web3.to_wei(40_000, "ether")
 DEAUTHORIZATION_DURATION = 60 * 60 * 24 * 60  # 60 days in seconds
 
+STAKE_INFO_OPERATOR_SLOT = 0
 
-def test_authorization_increase(accounts, threshold_staking, pre_application):
+
+def test_authorization_increase(accounts, threshold_staking, pre_cbd_application, stake_info):
```

Comment:
Do we want to just call `pre_cbd_application` -> `taco_application`? This and other test files.

Reply:
✔️